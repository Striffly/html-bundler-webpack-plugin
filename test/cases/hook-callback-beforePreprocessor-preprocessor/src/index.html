<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <link href="./style.css" rel="stylesheet" />
</head>
<body>
  <h1>Hello World!</h1>
  <!-- REPLACEME_beforePreprocessor_hook -->
  <!-- REPLACEME_beforePreprocessor_callback -->
  <!-- REPLACEME_preprocessor_hook -->
  <!-- REPLACEME_preprocessor_callback -->
</body>
</html>