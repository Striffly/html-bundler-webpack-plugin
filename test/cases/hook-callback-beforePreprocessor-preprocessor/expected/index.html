<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <link href="style.47f4da55.css" rel="stylesheet" />
</head>
<body>
  <h1>Hello World!</h1>
  <p>beforePreprocessor hook - OK</p>
  <p>beforePreprocessor callback - OK</p>
  <p>preprocessor hook - OK</p>
  <p>preprocessor callback - OK</p>
</body>
</html>